src/main/javascript/src/resources/components/header/header.js

```diff
@@ -17,11 +17,25 @@ export class Header {
             }
         })
         this.showNavOptions = false;
+        this.aurelia = aurelia
     }
 
     scroll_positionChanged(newValue, oldValue) {
         if (newValue > 90) {
             this.showNavOptions = false;
         }
     }
+
+    onSignOut() {
+		let auth2 = gapi.auth2.getAuthInstance();
+		let parent = this;
+		auth2.signOut().then(function () {
+			console.log('User signed out.');
+			parent.aurelia.setRoot(PLATFORM.moduleName('pages/login/login'));
+		});
+	}
+
+	isSignedIn(){
+		return true;
+	}
```

Comment:
As far as I can tell, this is only used in the header component, yeah? `if.bind="isSignedIn()"` which will always be true. I'd assume you're looking to do something more with this to check the session, which is why it's like this for now, yeah?

No change required

Reply:
yep, that's a placeholder until I get some kind of session table in the back-end